### PR TITLE
chore(opentelemetry): update trace example to latest released APIs

### PR DESCRIPTION
**Pull request by briandealwis**
Update the OpenTelemetry example from OT SDK v0.11.0 to v0.13.0.
I needed to explicitly shutdown the Cloud Trace exporter to cause it to flush any pending spans to Cloud Trace.

I encountered this code when trying to follow the example at:

   https://cloud.google.com/trace/docs/setup/go-ot

### CODE REVIEW

File: opentelemetry/trace/main.go [hunk 1]
```diff
@@ -63,6 +60,7 @@ func main() {
 
 		return nil
 	}(ctx)
+	exporter.Shutdown(ctx) // flush any pending spans
```
Comment (tbpg) [inline opentelemetry/trace/main.go:63]: Should this be called with `defer`?

**Review by tbpg — COMMENTED** [review]
Thank you!

File: opentelemetry/trace/main.go (hunk 1 above)
Comment (briandealwis) [reply opentelemetry/trace/main.go]: I had wondered, but `ctx` was defined further below.  Hmm, I'm not familiar with this code, but I wonder if we're better using the exporter options?

Comment (briandealwis) [reply opentelemetry/trace/main.go]: I don't see [any combination of bundler options][opts] that can be set to disable the bundling.

[opts]: github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace/trace.go

**Review by tbpg — APPROVED** [review]
Thank you!

File: opentelemetry/trace/main.go (hunk 1 above)
Comment (tbpg) [reply opentelemetry/trace/main.go]: Defer looks good to me. Thank you for checking!

**Review by tbpg — APPROVED** [review]

**Pull request merged**